src/main/java/com/easypost/http/Requestor.java

```diff
@@ -566,6 +564,7 @@ protected static void handleAPIError(String rBody, final int rCode) throws EasyP
         }
     }
 
+    @Generated // Exclude from the jacoco test coverage
```

Comment:
Nit: I would imagine there's a more appropriate annotation to exclude a code section from unit test coverage, instead of piggybacking off the Generated annotation. Won't block.

Reply:
Jacoco released this under 0.8.2 new features: https://github.com/jacoco/jacoco/releases/tag/v0.8.2